bin/k8s-deploy

```diff
@@ -203,7 +203,7 @@ do
   # this: https://github.com/kubernetes/kubernetes/issues/25238
 
   # If we can match 'replicas: hpa' in te deployment file
-  if [[ -n $( awk -F: '/[rR][eE][pP][lL][iI][cC][aA][sS][[:space:]]*:[[:space:]]*+[hH][pP][aA]/' $DEPLOYMENT_FILE ) ]]
+  if [[ -n $( awk -F: '/[rR][eE][pP][lL][iI][cC][aA][sS] *: +[hH][pP][aA]/' $DEPLOYMENT_FILE ) ]]
```

Comment:
The k8s spec isn't case-insensitive so this is overkill anyway. I think we should just change this to `/replicas: hpa/` and be done.

Reply:
I went ahead and did that in #112. If you're ok with it there we can close this.